packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -78,13 +78,14 @@ export const styles = (theme) => ({
     padding: '4px 8px',
     fontSize: theme.typography.pxToRem(11),
     maxWidth: 300,
+    margin: '2px',
```

Comment:
What does this change address?

Reply:
It's a small detail, it fixes this visual issue:

<img width="103" alt="Capture d’écran 2020-11-11 à 10 36 20" src="https://user-images.githubusercontent.com/3165635/98794818-c8606c00-2409-11eb-8826-55d9e50d73f2.png">